common/engine/keyboardprocessor/tests/unit/kmx/000 - null keyboard.kmn

```diff
@@ -1,5 +1,5 @@
 c Description: Tests null keyboard 
-c keys: [K_A][RALT K_B][SHIFT K_C]
+c keys: [K_A][LALT K_B][SHIFT K_C]
```

Comment:
Why this change? RALT was chosen intentionally.

Reply:
Note that the processor should give no output for either RALT or LALT here. Which is the intent of this test. As there appear to be differences between Linux and Windows for AltGr on US English (US English is a base assumption for the tests), we need to document those.